$OpenBSD: patch-pkcs1-decrypt_c,v 1.1 2015/05/01 16:03:35 ajacoutot Exp $

pkcs1-decrypt.c:46: warning: incompatible implicit declaration of built-in function 'alloca'

--- pkcs1-decrypt.c.orig	Fri May  1 18:01:00 2015
+++ pkcs1-decrypt.c	Fri May  1 18:01:31 2015
@@ -26,6 +26,7 @@
 # include "config.h"
 #endif
 
+#include <stdlib.h>
 #include <string.h>
 
 #include "pkcs1.h"
